### PR TITLE
In MySQLMetadataManager, set configurable timeout for executing SQL statements 

### PR DESCRIPTION
**Pull request by sophiayao**
### Summary
Currently, we allows at most 1 second for executing SQL queries to get table columns and primary key. This PR makes this duration configurable by its caller, and sets the default the same as before.

### CODE REVIEW

**Comment by sophiayao** [comment]
@ericwalton-asana  How do you check this code compiles? Is there a way to test it?

File: mypipe-api/src/main/scala/mypipe/mysql/MySQLMetadataManager.scala [hunk 1]
```diff
@@ -72,7 +72,7 @@ class MySQLMetadataManager(hostname: String, port: Int, username: String, passwo
       val mapCols = getTableColumns(db, table, dbConn(0))
```
Comment (sophiayao) [inline mypipe-api/src/main/scala/mypipe/mysql/MySQLMetadataManager.scala:72]: on line 68, database connection times out after 5 seconds. Eric and I think it's generous enough, but open to changing it to be configurable as well

Comment (ramatevish) [reply mypipe-api/src/main/scala/mypipe/mysql/MySQLMetadataManager.scala:72]: We're seeing this timing out on invalidators, unfortunately (see https://app.scalyr.com/events?filter=message+contains+%22getTableColumns%22&startTime=7d&teamToken=CoX7TpFAqLjqYwbafYKNKQ--&log=%2Fvar%2Flog%2Fworldst-prod_invalidator%2Finvalidator.log for both the 1 second and the 5 second timeouts)

**Comment by arvindvjkumar** [comment]
Ok just paired with @sophiayao on this, should be good to review

File: mypipe-api/src/main/scala/mypipe/mysql/MySQLMetadataManager.scala (hunk 1 above)
Comment (sophiayao) [reply mypipe-api/src/main/scala/mypipe/mysql/MySQLMetadataManager.scala:72]: @arvindvjkumar  Just made this configurable as well, do you mind reviewing and testing locally again?

**Comment by sophiayao** [comment]
@arvindvjkumar  when you pull mypipe into codez, probably want to make sure this change is included

**Comment by arvindvjkumar** [comment]
> @arvindvjkumar when you pull mypipe into codez, probably want to make sure this change is included

Ya I haven't started on that yet since I've been dealing with the follow-up for the NPE issue. I'm not going to start doing it till all of the in-flight mypipe changes are out (just to avoid adding more factors to a potentially risky migration)

File: mypipe-api/src/main/scala/mypipe/mysql/MySQLMetadataManager.scala (hunk 1 above)
Comment (arvindvjkumar) [reply mypipe-api/src/main/scala/mypipe/mysql/MySQLMetadataManager.scala:72]: I'll bump this higher + validate test in sand later today

**Pull request merged**